### ISSUE TITLE
Input_2 shape error

### ISSUE THREAD

**Issue by grassknoted** [grassknoted/CDAE4InfoExtraction#5]
There is a small issue with the decoding part of the model due to which the training fails, here it is:

![screenshot from 2018-12-23 05-16-56](https://user-images.githubusercontent.com/14085016/50379547-306d0400-0672-11e9-9f45-ea9f9b07eff7.png)

The error is caused because of the layer defined on line 100 of ```capsulenet.py```. 

Fixing this error (in a hacky way) leads to another error which is:
![screenshot from 2018-12-23 05-30-00](https://user-images.githubusercontent.com/14085016/50379604-d2d9b700-0673-11e9-93c0-624c32a9eb79.png)

In a nutshell, the model expects a ```(12, 1)``` vector, but instead a ```(13, 1)``` vector is being passed. There are 13 classes split amongst the 3 domains, so the input layer must be changed.

**Comment by akhil-reddy** [comment]
Resolved
